cppseminar/presentation/Services/AuthenticationService.cs

```diff
@@ -7,6 +7,8 @@
 using Microsoft.AspNetCore.Authentication.OAuth;
 using Microsoft.Extensions.Configuration;
 using presentation.Model;
+using System.Text;
```

Comment:
Neviem ci mi to github zobrazuje spravne, ale toto vyzera ako keby si pridal nejake veci navyse do suboru, s ktorym si nic nerobil